static/override-sab.css

```diff
@@ -0,0 +1,4 @@
+body {
```

Comment:
SAB stopped excluding margin-top, margin-bottom which was a bit of a hack.  We should exclude it from inside the PWA.